libtiledbsoma/src/pyapi/libtiledbsoma.cc

```diff
@@ -322,18 +408,113 @@ PYBIND11_MODULE(libtiledbsoma, m) {
                     uintptr_t arrow_array_ptr = (uintptr_t)(&arrow_array);
 
                     // Call array._export_to_c to get arrow array and schema
+                    //
+                    // If ever a NumPy array gets in here, there will be an
+                    // exception like "AttributeError: 'numpy.ndarray' object
+                    // has no attribute '_export_to_c'".
+                    // TODO: try/catch with something more helpful of the form
```

Comment:
Did you plan to address the TODO in this PR?

Reply:
Probably not now that I understand more

I'll take this out